sources/packages/web/src/components/aest/students/modals/AddManualOverawardDeduction.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <v-form ref="addManualOverawardForm">
+    <modal-dialog-base title="Add manual record" :showDialog="showDialog">
+      <template #content>
+        <error-summary :errors="addManualOverawardForm.errors" />
+        <p class="label-value">
+          Add a record to capture that the student paid back their loans through
+          the National Student Loans Service Centre (NSLSC).
+        </p>
+        <v-select
+          label="Award"
+          density="compact"
+          :items="awardTypeItems"
+          v-model="formModel.awardValueCode"
+          variant="outlined"
+          :rules="[(v) => checkNullOrEmptyRule(v, 'Award Type')]"
+        />
+        <v-text-field
+          hide-details="auto"
+          density="compact"
+          label="Add the dollar amount"
+          v-model="formModel.overawardValue"
+          variant="outlined"
+          type="number"
+          prefix="$"
+          :rules="[
+            (v) => checkNullOrEmptyRule(v, 'Overaward deduction amount'),
+            (v) =>
+              numberRangeRule(
+                v,
+                -MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE,
+                MONEY_VALUE_FOR_UNKNOWN_MAX_VALUE,
+                'Overaward deduction amount',
+              ),
+          ]"
+        />
+      </template>
+      <template #footer>
+        <check-permission-role :role="allowedRole">
+          <template #="{ notAllowed }">
```

Comment:
I am aware that it works and we used it before in other places but I am now getting the below error. Has anyone else faced the same? I am using `ESLint v2.4.0`, `Vetur v0.37.3`, `Prettier v9.10.4`
![image](https://user-images.githubusercontent.com/61259237/220992792-2984ec72-b2f0-4b49-a501-a31768c44eda.png)

Reply:
I am not getting it
![image](https://user-images.githubusercontent.com/77353155/221024668-ddc1137f-ccb8-459b-b929-9354c7aa4e70.png)
 
 I am using ESLint v2.4.0, Vetur v0.37.3, Prettier v9.10.4ttps://user-images.githubusercontent.com/77353155/221025392-9ad10d3b-c0d0-498c-af3a-91dd00f6d150.png)

Reply:
I am also getting the same error everywhere #="{ notAllowed }" is used. 
But it doesn't show up in the npm run serve console.

![image](https://user-images.githubusercontent.com/54600590/221029757-b6fb9cd7-fad8-41f9-b174-41a15632e7f3.png)

Reply:
@ann-aot did you check the extensions versions? Are they the same ones that you are using?

Reply:
ya @andrewsignori-aot Iam using the same ESLint v2.4.0, Vetur v0.37.3, Prettier v9.10.4

Reply:
I missed the details on your comment @ann-aot. 
Either way, I would prefer to stop using the syntax.